Modules/_sqlite/cursor.h

```diff
@@ -64,7 +64,7 @@ PyObject* pysqlite_cursor_fetchall(pysqlite_Cursor* self, PyObject* args);
 PyObject* pysqlite_noop(pysqlite_Connection* self, PyObject* args);
 PyObject* pysqlite_cursor_close(pysqlite_Cursor* self, PyObject* args);
 
-int pysqlite_cursor_setup_types(void);
+void pysqlite_cursor_setup_types(void);
```

Comment:
https://github.com/python/cpython/blob/f7b1e461567e5e3fa3ba46f589d9edc1b45b2dd0/Objects/typeobject.c#L5454

Since PyTypeReady check wheather the type is ready status.
You don't have to change the init functions.

Reply:
Please revert the relevant changes.

Reply:
Thanks. The `pysqlite_*_setup_types()` functions could still need a clean-up after this. I guess that would call for a separate issue.

Reply:
Fixed in commit d96c5e40a9286eeaeddedbdd1d6276fb0eb1a142